packages/editor-ui/src/components/RunData.vue

```diff
@@ -1290,6 +1298,18 @@ export default defineComponent({
 				saveAs(blob, fileName);
 			}
 		},
+		async downloadJsonData() {
+			const inputData = this.getNodeInputData(this.node, this.runIndex, this.currentOutputIndex);
+
+			const bufferString = `data:application/json;base64,${btoa(
+				JSON.stringify(inputData, null, 2),
+			)}`;
+
+			const fileName = this.node!.name.replace(/[^\w\d]/g, '_');
+
+			const blob = await fetch(bufferString).then(async (d) => d.blob());
```

Comment:
You can simplify this to 
```suggestion
			const fileName = this.node!.name.replace(/[^\w\d]/g, '_');
			const blob = new Blob([JSON.stringify(inputData, null, 2)], { type : 'application/json' });
```